pyam/utils.py

```diff
@@ -169,6 +169,10 @@ def format_data(df):
             df['scenario'] = scen.apply(
                 lambda s: '-'.join(s.split('-')[1:]).strip())
 
+    # reset the index if meaningful entries are included there
+    if not list(df.index.names) == [None]:
```

Comment:
here I think you could do just `if df.index.names:`

Reply:
just tried this, seems that this returns `True` even if no names are given